polyfill/lib/casts.mjs

```diff
@@ -189,8 +189,6 @@ export function CastDuration(arg) {
       return Duration.fromString(arg);
     } catch (ex) {}
   }
-  if ('bigint' === typeof arg) return new Duration(0, 0, 0, 0, 0, 0, 0, 0, arg);
-  if (Number.isFinite(+arg)) return new Duration(0, 0, 0, 0, 0, 0, +arg, 0, 0);
```

Comment:
Should we make a fromMiliseconds API to subsume this?

Reply:
That occurred to me too, yes. Let's do it.